src/app/malaria-101/activity-2/activity-2.component.ts

```diff
@@ -72,17 +82,31 @@ export class DragdropComponent implements OnInit {
      */
 
     addTobox($event: any, $box: String) {
-        this.reset();
-        if ($event.dragData.value === $box) {
-          this.score += this.POINTS_PER_CORRECT_ANSWER;
+      this.reset();
+      let pointsPerCorrectAnswer = this.POINTS_PER_CORRECT_ANSWER;
```

Comment:
rename this (pointsPerCorrectAnswer) local variable to 'score' or 'points' or 'cumulativePoints'. It is confusing if it has the same name as the constant (POINTS_PER_CORRECT_ANSWER). (upper case or lower case does not help)